src/components/Media/index.js

```diff
@@ -66,8 +66,12 @@ export default class Media extends PureComponent {
   componentDidMount() {
     if (this.props.onDimensions && this.dimensionElement)
       this.props.onDimensions({
-        width: this.dimensionElement.clientWidth,
-        height: this.dimensionElement.clientHeight,
+        width:
```

Comment:
Thanks a lot. Nice finding. Can you please add comment about Firefox near this code-block, to explain why this code is needed, for future people who will look into it - basically what you wrote in PR description